registry/server/api_topics_test.go

```diff
@@ -55,17 +56,40 @@ func TestListTopics(t *testing.T) {
 
 	tests := map[int]*pb.TopicRequest{
 		0: {},
-		1: {Name: "test_topic"},
-		2: {Tag: []string{"partitions:5"}},
+		1: {Spanning: true, Name: "test_topic"},
```

Comment:
I wonder if it wouldn't be clearer to provide a `SetBrokerMeta` function on the stub, and have full test set here, e.g. a tuple of `map[int]BrokerMeta`, `TopicReqest`, and the expected `TopicResult`? Basically (table-driven tests)[https://github.com/golang/go/wiki/TableDrivenTests].

Reply:
Yeah this is basically a "tables driven test" plural; sometimes I do it for readability when there's large structs, but that's not really the case here